### PR TITLE
data: waellerbe

### PR DESCRIPTION
**Pull request by waellerbe**
Updates for my JSON file
- BioDrop profile updates

## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

**Review by kumarsonsoff3 — DISMISSED** [review]
Please validate your json at [jsonlint.com](https://jsonlint.com)

File: data/waellerbe.json [hunk 1]
```diff
@@ -1,11 +1,18 @@
 {
   "name": "William A Ellerbe",
-  "bio": "Prompt Engineering Consultant with a passion for Open Source projects",
-  "links": [
+  "bio": "Open Source | Prompt Engineering | Proud member of **EddieHub**",
+"links": [
 {
+      "group": Socials Primary
```
Comment (SaraJaoude) [inline data/waellerbe.json:6]: adding quotation marks and comma:
```suggestion
      "group": "Socials Primary",
```

File: data/waellerbe.json [hunk 2]
```diff
@@ -1,11 +1,18 @@
...
+"links": [
 {
+      "group": "Socials Primary",
       "name": "Follow me on LinkedIn",
       "url": "https://www.linkedin.com/in/williamaellerbe/",
       "icon": "FaLinkedin"
-    }
+    },
+    {
+      {
```
Comment (SaraJaoude) [inline data/waellerbe.json:12]: removing bracket:
```suggestion
```

File: data/waellerbe.json [hunk 3]
```diff
@@ -1,11 +1,17 @@
...
       "name": "Follow me on LinkedIn",
       "url": "https://www.linkedin.com/in/williamaellerbe/",
       "icon": "FaLinkedin"
-    }
+    },
+    {
+      "group": "Socials Primary",
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/waellerbe",
+      "icon": "FaGithub",
```
Comment (SaraJaoude) [inline data/waellerbe.json:15]: removing comma and adding closing bracket:
```suggestion
      "icon": "FaGithub"
   }
```

**Review by SaraJaoude — APPROVED** [review]
@waellerbe I have made the necessary corrections to the json and can now merge this PR.

**Pull request merged**